### PR TITLE
#2 Move all tests from unittest to py.test format

### PR DESCRIPTION
**Pull request by qaivanpleten**
@extsoft check please. also check question in #2

### CODE REVIEW

File: sp_at/test_cases/check_about_us_page.py [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
-
-    def test_buttons(self):
-        general_action = GeneralActions(self.driver)
-        about_element = AboutPageElements(self.driver)
-        general_action.open_page_by_url('http://console.dev.sonikpass.com/#about')
-        time.sleep(2)
-        general_action.click_on_button(about_element.open_positions_button())
-        time.sleep(2)
-        general_action.check_url('http://console.dev.sonikpass.com/#careers')
+from sp_at.actions.setup_class import fixture_webdriver
```
Comment (extsoft) [inline sp_at/test_cases/check_about_us_page.py]: @qaivanpleten this fixture has to be located in `conftest.py` file within `sp_at/test_cases`. Then you won't need to import it. Please update.

**Review by extsoft — CHANGES_REQUESTED** [review]

File: sp_at/test_cases/check_about_us_page.py (hunk 1 above)
Comment (qaivanpleten) [reply sp_at/test_cases/check_about_us_page.py]: @extsoft 
I'll do it, but could you explain. what's the reason?

File: sp_at/ignore_these_cases/.cache/v/cache/lastfailed [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (extsoft) [inline sp_at/ignore_these_cases/.cache/v/cache/lastfailed]: @qaivanpleten please remove `.cache` directory from git.

**Review by extsoft — CHANGES_REQUESTED** [review]
@qaivanpleten just one comment here.

**Comment by extsoft** [comment]
@qaivanpleten also, please rebase `master` to this branch.

File: sp_at/test_cases/check_about_us_page.py (hunk 1 above)
Comment (extsoft) [reply sp_at/test_cases/check_about_us_page.py]: Because of `pytest` automatically discover all fixtures within `conftest.py`.

Comment (extsoft) [reply sp_at/test_cases/check_about_us_page.py]: @qaivanpleten Did you run the tests successfully after removing the import?

**Review by extsoft — APPROVED** [review]

**Pull request merged**